src/integrationTest/java/uk/gov/hmcts/reform/sendletter/TestApplication.java

```diff
@@ -0,0 +1,13 @@
+package uk.gov.hmcts.reform.sendletter;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+
+@SpringBootApplication
+@SuppressWarnings("HideUtilityClassConstructor") // Spring needs a constructor, its not a utility class
+public class TestApplication {
```

Comment:
is this used / adding value?

Reply:
There has to be an application otherwise integration tests are not working - `IllegalState`